server/chat-commands.js

```diff
@@ -3796,6 +3796,60 @@ const commands = {
 	},
 	importinputloghelp: [`/importinputlog [inputlog] - Starts a battle with a given inputlog. Requires: + % @ & ~`],
 
+	rejectdraw: 'offertie',
+	rejecttie: 'offertie',
+	acceptdraw: 'offertie',
+	accepttie: 'offertie',
+	offerdraw: 'offertie',
+	offertie(target, room, user, connection, cmd) {
+		const battle = room.battle;
+		if (!battle) return this.errorReply("Must be in a battle room.");
+		if (!Config.allowrequestingties) {
+			return this.errorReply("This server does not allow offering ties.");
+		}
+		if (!this.can('roomvoice', null, room)) return;
+		if ((cmd === 'accepttie' || cmd === 'rejecttie') && !battle.players.some(player => player.wantsTie)) {
+			return this.errorReply("No other player is requesting a tie right now. It was probably canceled.");
+		}
+		const player = battle.playerTable[user.userid];
+		if (cmd === 'rejecttie' && battle.players.some(player => player.wantsTie)) {
+			if (!player) {
+				return this.errorReply("Must be a player to reject ties.");
+			}
+			if (player.wantsTie) player.wantsTie = false;
+			for (const player of battle.players) {
+				player.sendRoom(Chat.html`|uhtmlchange|offertie|(Tie offer rejected)`);
+			}
+			return this.add(`${user.userid} rejects to tie this battle`);
```

Comment:
```suggestion
			return this.add(`${user.userid} rejects tying this battle`);
```

`declines to tie this battle` also works, but might sound too formal?

Reply:
I feel like using 'rejects' is better because the command name is `/rejecttie` and not `/declinetie`